.gitignore

```diff
@@ -68,3 +68,4 @@ typings/
 .DS_Store
 
 # NQL Custom
+test.db
```

Comment:
@kirrg001 will make this a `*.db` rule insteady and push straight to master. any objections? :hear_no_evil:

Reply:
landed in master with 41d3d051ca6a03056b7c3f6af89c5963dcadfd50

Reply:
👍